### ISSUE TITLE
Test against WordPress 5.8

### ISSUE THREAD

**Issue by jeffpaul** [10up/post-finder#34]
**Is your enhancement related to a problem? Please describe.**
Once WordPress 5.8 is released, we'll want to test Post Finder to see if any incompatibility issues arise.

**Describe the solution you'd like**
- [x] test Post Finder on WordPress 5.8
- [ ] open issues for any incompatibilities noted in testing
- [ ] resolve issues identified in testing
- [x] bump "tested up to" version
- [ ] if code changes needed due to incompatibilities, ship a plugin release

**Designs**
n/a

**Describe alternatives you've considered**
none

**Additional context**
n/a

**Comment by BBerg10up** [comment]
Hey @jeffpaul, I wasn't sure which branches to test so I did a quick sanity check on::
- Trunk
- Master
- 0.4.0
- Develop

I didn't encounter any compatibility issues. I compared the functionality from 5.8 vs. 5.4 (the last tested version), and didn't see any differences.

I tried to bump the tested up to version, but may need assistance.

- Develop: Access denied
- Trunk: https://github.com/10up/post-finder/commit/422f4ccd53870c2c5bc55267d5a7760671298325
- Master: https://github.com/10up/post-finder/commit/4bc4c7fbad9936896137efb94af761f7eb2fd92c
- Release 0.4.0: https://github.com/10up/post-finder/commit/383f8edc83b23fb234600835375d431b406ad5f8
